### ISSUE TITLE
Message Format Uniformity

### ISSUE THREAD

**Issue by ShendMikullovci** [serge-web/serge#1637]
I was wondering why some messages differ from others in terms of format:

![Messages in-game](https://user-images.githubusercontent.com/85491794/175826991-9b82d08f-0106-4697-8fd0-d760eee08ad6.png)

![Messages in-game 2](https://user-images.githubusercontent.com/85491794/175827000-05afd538-ec35-4b78-9f1a-80da2de2414a.png)

If there's no specific reason, I suggest we adapt the 2nd design for all messages across channels.

**Comment by IanMayo** [comment]
They are two different types of channel.  The top one is a chat-channel, the lower one is standard channel (where the player can choose the template).

They have these differences: chat is meant to be short snippets. So, they are not collapsible, and no templates are available.  They also fill up from the top, like instant messaging.

The standard channel tends to have larger messages, including templates.  They can be quite tall - so they start off collapsed.  They also fill up from the bottom, so they are in chronological order going down the page.

The above were all Céline's ideas - though we don't have to keep it that way.

**Comment by ShendMikullovci** [comment]
Yes, I understand. However, I don't think that the way we display messages in Strat Comms vs Red COA here leads the user to understand that difference. 
Why is the Force color on the right of the message for Strat Comms and on the left for Red COA for example? 

I think going for uniformity here would again lead the user's attention to the content of the message, rather than the look of the message.

**Comment by IanMayo** [comment]
Yes, good point.  In both channel styles the "sending force" color should be on the outside (left for messages from others, right for messages from own force).
